lucille-core/src/main/java/com/kmwllc/lucille/core/Document.java

```diff
@@ -652,7 +670,7 @@ default void setNestedJson(String name, JsonNode value) {
   }
 
   // if the given segment is an integer, it will create a new ArrayNode, otherwise an ObjectNode
-  private JsonNode createNewNode(String segment) {
+  static private JsonNode createNewNode(String segment) {
```

Comment:
"private static" is our convention, instead of "static private" (even though they mean the same)